<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTVTK_MSVEApplication</class>
 <widget class="QMainWindow" name="QTVTK_MSVEApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleView</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnDifference">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Difference</string>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidgetLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>361</width>
      <height>439</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidgetRight">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>361</width>
      <height>440</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="beforeRightNippleBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Right Nipple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beforeNavalBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Naval</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beforeLeftNippleBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>530</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Left Nipple</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="afterRightNippleBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>530</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Right Nipple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="afterLeftNippleBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Left Nipple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="afterNavalBtn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Naval</string>
    </property>
   </widget>
  </widget>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/filesave.png</normaloff>:/Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
